homeassistant/components/tado/climate.py

```diff
@@ -767,3 +800,20 @@ def _control_hvac(
             vertical_swing=vertical_swing,  # api defaults to not sending verticalSwing if swing not None
             horizontal_swing=horizontal_swing,  # api defaults to not sending horizontalSwing if swing not None
         )
+
+    def _does_current_hvac_mode_have_setting(self, setting: str) -> bool:
```

Comment:
```suggestion
    def _current_hvac_mode_have_setting(self, setting: str) -> bool:
```
`does` sounds strange
```suggestion
    def _is_valid_setting_for_hvac_mode(self, setting: str) -> bool:
```
works maybe better